woodwork/table_accessor.py

```diff
@@ -841,6 +842,23 @@ def mutual_information(self, num_bins=10, nrows=None, include_index=False, callb
         mutual_info = self.mutual_information_dict(num_bins, nrows, include_index, callback)
         return pd.DataFrame(mutual_info)
 
+    def get_valid_mi_columns(self, include_index=False):
+        """Retrieves a list of columns from the DataFrame with valid Logical Types that support mutual
+        information. Logical Types that support mutual information are as follows:  Age, AgeFractional,
+        AgeNullable, Boolean, BooleanNullable, Categorical, CountryCode, Datetime, Double, Integer,
+        IntegerNullable, Ordinal, PostalCode, and SubRegionCode.
```

Comment:
@gsheni @tuethan1999 Should we be explicitly listing out the types here in the docstring?  Seems like something that could potentially become out of date.

Reply:
I think it's a minor thing. We can keep updating the docstring. 
We also have a [get_valid_mi_types](https://woodwork.alteryx.com/en/latest/generated/woodwork.utils.get_valid_mi_types.html#woodwork.utils.get_valid_mi_types) that we could potentially reference in the docstring.

Reply:
I also think we should probably reference [get_valid_mi_types](https://woodwork.alteryx.com/en/latest/generated/woodwork.utils.get_valid_mi_types.html#woodwork.utils.get_valid_mi_types) instead of listing out the types